### DISCUSSION TITLE
Initial control mode Manual ?

### DISCUSSION THREAD

**Discussion by maver1ck** [maca88/SmartBikeLights#31] [Q&A]
Hi,
I'm started to configure lights and I have one problem.
When I'm starting activity Initial Mode is OFF.  Is it because of my global filter on Timer State = Recording ?
But then after activity start mode should change.

My config:
```
1,1!Start:1H=3#0,1073754640#6,4!High:2:2:60:0Es600,r-600C]8.33333!Medium:2:3:60:0Es600,r-600C]5.55556!Low:1:4:60:0Es600,r-600!:1:0:0:0D=1#0,73535488#2,2!Night:1:6:60:0Es600,r-600!:1:7:0:0D=1#6,5:HL 1800!2,:-1,Off:0!1,High:2!1,Medium:3!1,Low:4!1,Day Flash:8#6,5:Varia 515!2,:-1,Off:0!1,Solid:4!1,Peloton:5!1,Day Flash:7!1,Night Flash:6#0::#0:0#123!:123!#B3122#7#3#0#0
```

**Comment by maca88** [comment]
Hi,

>  Is it because of my global filter on Timer State = Recording ?

The global filter that you set will only turn on the light (based on the light filters) when the activity is recording. In case you started the activity and is recoding (not paused) and the light is still OFF, then it is because the Timespan filter needs to get a GPS fix so that the data field can calculate the sunset/sunrise. Until the GPS fix is not acquired, the Timespan filter will always evaluate to false, which means that in you case the "Default mode" will be used (which is set to OFF for the headlight).

**Reply by maver1ck** [reply]
@maca88 Thanks for the answer.
I think I wrongly describe the problem.
Not only Initial Mode is OFF but the Initial Control Mode is Manual. And I think this is the problem. Changing to Smart makes everything working properly (at least the things I tested)

I attached screenshot of your data field after starting activity.
![20220608_081730](https://user-images.githubusercontent.com/4006010/172545809-5a41d5f9-b5f4-44f2-9331-c67664ac7863.jpg)

**Reply by maca88** [reply]
Once you select the Smart control mode (S), it should stay in this mode until you change the light mode by either using the data field light panel or the physical button on the light. Let me know in case the Smart control mode (S) does change to Manual (M) every time you start an activity, because that should not happen.

**Reply by yareckey** [reply]
Hi How did You come to that screen? On mine Edge 530 I can only see this 
![20221006_105909](https://user-images.githubusercontent.com/80855532/194282831-895f3a96-9b13-4fec-8c47-19a0412001ca.jpg)

**Reply by maca88** [reply]
Hi,

the light panel is available only for Edge touchscreen devices (e.g. Edge 1030), for Edge 530 you need to use the data field menu to manually switching light modes.

**Reply by yareckey** [reply]
Hi Thanks all is OK now. Good Job !

**Reply by yareckey** [reply]
Hi again I've tested my configuration in the field and I get one problem with Varia RTL515 even if it is set to DayFlash the light always uses same mode Solid

My Garmin is Edge 530, My lights are: Bontrager ION 200, Garmin Varia RTL 515

My intentions are to set lights to Smart mode for the whole 24 h (day & night):
Front light: 
- Day Flash during the day
- High during night
- High if I cycle faster or equal to 25 kph
Rear light
- Day Flash for the whole 24 h (day & night)

My configuration is 1,1!Lights:1E86340,86280#4587520,196641::16776960#4,4!Fast:1:1:0:0C}6.94444!Day:1:63:0:0Er0,s0!Night:1:1:0:0Er0,s0!:1:63:0:0D=1#0,73535488::16711680#3,2!All:2:7:0:0Er0,s0Es0,r0!:1:7:0:0D=1#6:Ion 200 RT!Off:0!High:1!Medium:2!Low:5!Day Flash:63!Night Flash:62#5:Varia 515!Off:0!Solid:4!Peloton:5!Day Flash:7!Night Flash:6#0::#1:1#0#B3121#2#3#0#0

**Reply by maca88** [reply]
Your configuration is not optimal and the "Night" group will never be triggered as it has the same filter as "Day" group. Here is the optimized and fixed configuration:

#4587520,196641::16776960#4,4!Fast:1:1:0:0C}6.94444!Day:1:63:0:0Er0,s0!Night:1:1:0:0Es0,r0!:1:63:0:0D=1#0,73535488::16711680#1,1!:1:7:0:0D=1#6:Ion 200 RT!Off:0!High:1!Medium:2!Low:5!Day Flash:63!Night Flash:62#5:Varia 515!Off:0!Solid:4!Peloton:5!Day Flash:7!Night Flash:6#0::#1:1#0#B3121#2#3#0#0

If the light stays in "Solid", which is not even in the configuration then this means that the lights are not is "Smart" mode. Make sure that both lights are is "Smart" mode by assuring that the "S" character is inside both lights. In order to change the control mode you need to use the data field menu, check here for more info: https://github.com/maca88/SmartBikeLights#changing-control-modes

**Reply by yareckey** [reply]
Hi, thx a lot for the help and explanation :)  I will test my new config in the field tomorrow but it looks OK now. 
Jarek
